src/parsec_client/operations/service_api_coverage.md

```diff
@@ -36,7 +36,7 @@ sections for details.
 | [PsaAsymmetricEncrypt](psa_asymmetric_encrypt.md)   | 🚫    | ✅           | ✅       | ✅       | ✅               | ❌                  |
 | [PsaAsymmetricDecrypt](psa_asymmetric_decrypt.md)   | 🚫    | ✅           | ✅       | ✅       | ✅               | ❌                  |
 | [PsaRawKeyAgreement](psa_raw_key_agreement.md)      | 🚫    | ✅           | ❌       | ❌       | ❌               | ✅                  |
-| [PsaGenerateRandom](psa_generate_random.md)         | 🚫    | ✅           | ❌       | ❌       | ✅               | ✅                  |
+| [PsaGenerateRandom](psa_generate_random.md)         | 🚫    | ✅           | ✅       | ❌       | ✅               | ✅                  |
```

Comment:
could you also update it for the TPM provider. It supports genRandom since https://github.com/parallaxsecond/parsec/pull/595

Reply:
Okay will update it.